locales/en/default.json

```diff
@@ -159,6 +159,13 @@
         "pageTitle": "User Activity Report results"
       }
     },
+    "user-management": {
+      "serviceName": "Administration",
+      "organisationsPage": {
```

Comment:
Similar to one of my previous comments, I think this needs to be singular, rather than pluralised at this point:

"organisationPage"
"Manage organisation"
"Organisation"

Reply:
Done